onbuild/entrypoint.sh

```diff
@@ -1,7 +1,47 @@
 #!/usr/bin/env bash
 
-echo "NOTE:"
-echo "THIS CONTAINER IS FOR DEVELOPMENT PURPOSES ONLY AND SHOULD NOT BE USED IN PRODUCTION"
-echo ""
+if [ "$1" == "bootstrap" ] ; then
+    echo "You can replace this entrypoint.sh script and customize it to fire eventstore and run"
+    echo "some commands on it to configure streams, projections and subscriptions thus having"
+    echo "a virgin eventstore instance ready to deploy with the right setup for your needs."
+    echo ""
+    echo ""
+    echo "Basicaly you just have to wrap you es-cli and/or curl setup commands around that "
+    echo "kind of script template:"
+    echo ""
+cat <<EOF
+#!/usr/bin/env bash
+set -e
+BASIC_AUTH=admin:changeit
+
+eventstored -SkipDbVerify 0<&- &>/dev/null &
+PID=$!
+NB_TRY=0;
+echo -n "INFO: wait for eventstore to be available"
+while ! curl -o /dev/null -sf http://localhost:2113/projections/continuous
```

Comment:
Not sure this is the best way to know when eventstore is ready to accept setup instructions, I'm ok for a better solution if any exist ;)